packages/core/FuzzedDataProvider.ts

```diff
@@ -401,6 +401,30 @@ export class FuzzedDataProvider {
 		return this.consumeString(this._remainingBytes, encoding);
 	}
 
+	/**
+	 * Consumes an array of `string` form the fuzzer input.
```

Comment:
- "form" should be "from"
- maybe `string` should be `string`s (like in the next line)